test/general/test_policies.pass.cpp

```diff
@@ -68,7 +68,9 @@ main()
 
     // make_device_policy
     test_policy_instance(make_device_policy<class Kernel_11>(q));
+#if _ONEDPL_LIBSYCL_VERSION < 60000
```

Comment:
I think it should be explained here (with reference to line 75), otherwise it looks that we just get rid of on test.

Reply:
Fixed